uv.lock

```diff
@@ -0,0 +1,587 @@
+version = 1
```

Comment:
This file should be removed

Reply:
Ohhh sorry, I used `uv` to install and run the tests and accidentally committed this T_T

Removed!